exchange/bitswap/network/interface.go

```diff
@@ -48,4 +48,7 @@ type Routing interface {
 
 	// Provide provides the key to the network
 	Provide(context.Context, key.Key) error
+
+	// Provide multiple keys to the network at the same time, sharing RPCs
+	ProvideMany(context.Context, []key.Key) error
```

Comment:
wonder if that should be a key channel, to allow providing from a single dag in one call, and then do the stack thing this way